### PR TITLE
Cross browser testing

### PR DESCRIPTION
**Pull request by hammerfunk**
So far only tested Chrome, Firefox and Safari
A lot of general bug and responsive fixes

### CODE REVIEW

File: app/assets/stylesheets/frontend/forms.scss [hunk 1]
```diff
@@ -871,6 +873,10 @@ input[type="file"] {
   }
 }
 
+.step-article .pagination li.submit {
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Avoid qualifying class selectors with an element.

File: app/assets/stylesheets/frontend/views/award_form.scss [hunk 2]
```diff
@@ -204,3 +208,40 @@ label > .visible-read-only,
...
+    .span-3 {
+      float: left;
+      position: relative;
+      width: 60%;
+      max-width: 200px;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+      -webkit-box-sizing: border-box;
+         -moz-box-sizing: border-box;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/views/award_form.scss]: Line should be indented 6 spaces, but was indented 9 spaces<br>Avoid vendor prefixes.

File: app/assets/stylesheets/frontend/views/award_form.scss [hunk 3]
```diff
@@ -204,3 +208,40 @@ label > .visible-read-only,
...
+              box-sizing: border-box;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/views/award_form.scss]: Line should be indented 6 spaces, but was indented 14 spaces

File: app/assets/stylesheets/frontend/views/award_form.scss [hunk 4]
```diff
@@ -204,3 +208,40 @@ label > .visible-read-only,
...
+        content: " ";
+      }
+
+      &:after {
+        clear: both;
+      }
+    }
+
+    .span-3 {
+      float: left;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/views/award_form.scss]: Properties should be ordered position, width, max-width, min-height, padding-right, padding-left, float, -moz-box-sizing, -webkit-box-sizing, box-sizing

File: app/assets/stylesheets/frontend/views/award_form.scss [hunk 5]
```diff
@@ -204,3 +208,40 @@ label > .visible-read-only,
...
+      position: relative;
+      width: 60%;
+      max-width: 200px;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+      -webkit-box-sizing: border-box;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/views/award_form.scss]: Avoid vendor prefixes.

File: app/forms/qae_2014_forms/innovation/innovation_step3.rb [hunk 6]
```diff
@@ -155,7 +156,7 @@ def innovation_step3
           classes "sub-question total-net-assets"
           sub_ref "C 5.3"
           required
-          context %{<p>As per your balance sheet. Total assets (fixed and current), less liabilities (current and long-term).}
+          context %{<p>As per your balance sheet. Total assets (fixed and current), less liabilities (current and long-term).</p>}
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step3.rb]: Line is too long. [130/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step3.rb [hunk 7]
```diff
@@ -187,7 +188,7 @@ def trade_step3
           by_year_condition :trade_commercial_success, "3 to 5", 3
           by_year_condition :trade_commercial_success, "6 plus", 6
           context %(
-            <p>If you haven't reached your latest year-end, please use estimates to complete this question.
+            <p>If you haven't reached your latest year-end, please use estimates to complete this question.</p>
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step3.rb]: Line is too long. [111/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb [hunk 8]
```diff
@@ -151,7 +152,7 @@ def development_step3
           classes "sub-question total-net-assets"
           sub_ref "C 5.3"
           required
-          context %{<p>As per your balance sheet. Total assets (fixed and current), less liabilities (current and long-term).}
+          context %{<p>As per your balance sheet. Total assets (fixed and current), less liabilities (current and long-term).</p>}
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb]: Line is too long. [130/100]

File: app/assets/stylesheets/frontend/views/award_form.scss [hunk 9]
```diff
@@ -204,3 +208,40 @@ label > .visible-read-only,
...
+
+    .span-3 {
+      position: relative;
+      width: 60%;
+      max-width: 200px;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+      float: left;
+         -moz-box-sizing: border-box;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/views/award_form.scss]: Line should be indented 6 spaces, but was indented 9 spaces<br>Avoid vendor prefixes.

File: app/views/layouts/application.html.slim [hunk 10]
```diff
@@ -40,8 +40,9 @@
   - unless landing_page?
     .header-proposition
       .content
-        span.if-no-js-hide
-          = link_to "Menu", "#proposition-links", class: "js-header-toggle menu"
+        - if current_admin || current_user
```
Comment (RuslanHamidullin) [inline app/views/layouts/application.html.slim]: maybe also need "current_assessor" ?
for:
```
- if current_admin || current_user
```

Comment (RuslanHamidullin) [inline app/views/layouts/application.html.slim]:
```
- if current_admin || current_user || current_assessor
```

**Comment by RuslanHamidullin** [comment]
@hammerfunk all looks good.
But need to confirm:
```
- if current_admin || current_user
```
Probably need:
```
- if current_admin || current_user || current_assessor
```
Does it?

**Comment by RuslanHamidullin** [comment]
Super Puper :+1:

**Pull request merged**